### ISSUE TITLE
Multi-line notifications are cut off

### ISSUE THREAD

**Issue by jotomo** [MilosKozak/AndroidAPS#653]
Notifications on the home screen have their last line cut off for longer (multi-line) messages

**Comment by AdrianLxM** [comment]
Is this still an issue?

**Comment by jotomo** [comment]
Yes, still an issue. Add
```
bus().post(new EventNewNotification(new Notification(999, gs(R.string.bolus_frequency_exceeded), Notification.NORMAL)));
```
to the end of `MainApp.onCreate` to reproduce on start:
![screenshot_1525433411](https://user-images.githubusercontent.com/1732305/39625797-723a94f4-4f9f-11e8-88cc-27c63beb9f6e.png)

**Comment by AdrianLxM** [comment]
![screenshot from 2018-05-05 21-48-13](https://user-images.githubusercontent.com/9692866/39667046-13e1fe96-50ae-11e8-9e09-de43cedf62b6.png)
It seems to always just cut off the last line.

**Comment by jotomo** [comment]
Fixed via #1037, thanks @AdrianLxM
